frontend/src/modules/entitieslist/components/Entity.tsx

```diff
@@ -14,9 +16,14 @@ type Props = {
     isReadOnlyEditor: boolean;
     isTranslator: boolean;
     locale: Locale;
-    search: string | null | undefined;
     selected: boolean;
     selectEntity: (...args: Array<any>) => any;
+    getSiblingEntities: Function;
+    parameters: NavigationParams;
+};
+
+type State = {
+    siblingsAreActive: Boolean;
```

Comment:
Nit: `areSiblingsActive` would be consistent with our boolean variable naming convention which starts with a  verb (is, are, has ...).